src/components/generic/fund-card.tsx

```diff
@@ -95,7 +96,7 @@ const FundCard: React.FC<Props> = ({ fund }) => {
             <View style={ styles.cardRow }>
               <Icon name="calendar" size={ 12 } color={ color }/>
               <Text style={ styles.lastUpdated }>
-                { priceDate?.toLocaleDateString() }
+                { priceDate && moment(priceDate).format("DD.MM.YYYY") }
```

Comment:
This date format method is used multiple times in the codes, so maybe it could be generalized into  DateUtils.formatDate?